packages/eventual-send/test/test-types.js

```diff
@@ -2,10 +2,10 @@ import { test } from './prepare-test-env-ava.js';
 
 import { E } from './get-hp.js';
 
-/** @template T @typedef {import('../src/index').ERef<T>} ERef */
+/** @template T @typedef {import('../src/exports').ERef<T>} ERef */
```

Comment:
Should this test use the fully qualified name of the package? 
```suggestion
/** @template T @typedef {import('@endo/eventual-send').ERef<T>} ERef */
```

Reply:
While I was at it, I avoided the typedef-import antipattern.